Control coverage/Feature/CDN.md

```diff
@@ -62,3 +62,52 @@ One of the following conditions is met:
 properties.isHttpAllowed, properties.isHttpsAllowed, properties.deliveryPolicy.rules
  <br />
 
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial 
+
+### Display Name 
+[Trial] Front Door should have Approved Minimum TLS version
+
+### Rationale 
+TLS provides privacy and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions. 
+
+### Control Spec 
+
+> **Passed:** 
+>- For all custom domain, the Minimun TLS Version is set to TLS 1.2 or greater than that.
```

Comment:
IF there are no custom domain then also control will be in passed state, can we add that as well.